### PR TITLE
Update use-NDI-in-meetings.md

### PR DESCRIPTION
**Pull request by damarcin**
Small changes of doc file

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/use-NDI-in-meetings.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ NDI is limited to a local network and should only be considered a part of the pr
 
 NDI requires two steps to be turned on for a user.
 
-1. The tenant admin must enable the feature flag enableStreamingCallsOverNdi.
+1. The tenant admin must enable 'AllowNDIStreaming' property in CsTeamsMeetingPolicy.
```
Comment (JohanFreelancer9) [inline Teams/use-NDI-in-meetings.md:38]:
```suggestion
1. The tenant admin must enable the 'AllowNDIStreaming' property in CsTeamsMeetingPolicy.
```

Comment (damarcin) [reply Teams/use-NDI-in-meetings.md]: @JohanFreelancer9, thanks for your suggestions, please let me know if anything else is needed in order to complete this PR

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by cichur** [comment]
I've asked the SME to verify the changes. :-)

**Pull request merged**